### ISSUE TITLE
Why is `disable_monkey_patching` hidden?

### ISSUE THREAD

**Issue by lemairecarl** [comet-ml/issue-tracking#321]
For many learning contexts, logging loss at every step is not appropriate. In my situation, an epoch only takes a few seconds, so it makes more sense to log the loss at every step. However, if I try to log the loss myself, the monkey patching seems to interfere, and the resulting loss plot(s) are confusing. (I'm using PyTorch, btw.)

Could you please consider adding an argument to the `Experiment` constructor for disabling monkey patching? A parameter in `.comet.config` would also be nice.

Thanks! I love Comet.ml! :)

**Comment by dsblank** [comment]
@lemairecarl Perhaps you are looking for `auto_metric_logging=False` ? Also, what do you mean "A parameter .comet.config"? Thanks! We love you too :)

**Comment by lemairecarl** [comment]
Oops! I meant that a parameter inside `.comet.config` would be nice for disabling monkey patching.

At this time, I prefer not having Comet adding hooks to my model.

Thanks for pointing out `auto_metric_logging`. Would you mind tweaking the description of this argument in the docstring? Right now it reads:

```
auto_metric_logging: Default(True) - allows you to enable/disable metrics logging
```

which is confusing. I suggest rewording this as: "allows to enable/disable **automatic** metrics logging (works by adding hooks/callbacks to your model)" or something.

**Comment by dsblank** [comment]
Good idea... will do! Thanks!

**Comment by dsblank** [comment]
Done! Will come out in the next doc release. Thanks again!
